furnace.c

```diff
@@ -162,18 +165,38 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
     char msg[16];
     const size_t msg_len = snprintf(msg, sizeof(msg), "temp = %d\r\n", ctx->pilot.des_temp);
     feedback(msg, msg_len);
+  } else if (sscanf(buffer, "log %s %u", &str_arg, &arg) == 2) {
+    if(arg >= 2) {
+      const char msg[] = "log value too big!\r\n";
+      const size_t msg_len = sizeof(msg)-1;
+      feedback(msg, msg_len);
+    }
+    set_log(str_arg, arg, &ctx->log_bits);
+  } else if (strncmp(buffer, "log\n", 4) == 0) {
+    char msg[16];
+    const size_t msg_len = snprintf(msg, sizeof(msg), "%d\r\n", ctx->log_bits);
```

Comment:
Not veyr user friendly. Let's make it use strings to set and report enabled bits.